api/v1alpha1/envoyproxy_types.go

```diff
@@ -169,6 +173,14 @@ const (
 	EndpointRoutingType RoutingType = "Endpoint"
 )
 
+// ExtensionPolicySettings defines settings for EnvoyExtensionPolicies to be applied on the EnvoyProxy fleet.
+type ExtensionPolicySettings struct {
+	// DisableLuaValidation disables the Lua script validation for Lua EnvoyExtensionPolicies
+	// Default: false
+	// +optional
+	DisableLuaValidation *bool `json:"disableLuaValidation,omitempty"`
```

Comment:
imo, id put this directly at top level, can't think of which other fields go here
cc @envoyproxy/gateway-maintainers